jsapp/js/projects/projectsTable/sortableColumnHeader.tsx

```diff
@@ -0,0 +1,134 @@
+import React, {useState} from 'react';
+import classNames from 'classnames';
+import Icon from 'js/components/common/icon';
+import Button from 'jsapp/js/components/common/button';
+import type {ProjectsTableOrder} from './projectsTable';
+import {type ProjectFieldDefinition, type ProjectFieldName} from '../projectViews/constants';
+import styles from './projectsTableHeader.module.scss';
+import rowStyles from './projectsTableRow.module.scss';
+import KoboDropdown from 'jsapp/js/components/common/koboDropdown';
+
+interface SortableColumnHeaderProps {
+  styling: boolean;
+  field: ProjectFieldDefinition;
+  highlightedFields?: ProjectFieldName[];
+  orderableFields: ProjectFieldName[];
+  order: ProjectsTableOrder;
+  onChangeOrderRequested: (order: ProjectsTableOrder) => void;
+  onHideFieldRequested?: (fieldName: ProjectFieldName) => void;
+}
+
+
+export default function SortableColumnHeader(props: SortableColumnHeaderProps) {
```

Comment:
It might be better to include `Project` in this component name, since it currently only sorts project tables. (Making this component fully independent of projects would be a nice abstraction, but it seems like that would be a pain given how the props are set up.)